tests/functions/test_find_replace.py

```diff
@@ -50,6 +50,4 @@ def test_find_replace_regex(df_orders):
 @pytest.mark.functions
 def test_find_replace_regex_match_rases_error(df_orders):
```

Comment:
Can we change `rases` to `raises` in test name?

Reply:
Good catch.